Data/future/usa/margins/GC.csv

```diff
@@ -33,4 +28,20 @@ date,initial,maintenance
 20160226,4950,4500
 20160627,6050,5500
 20160701,6600,6000
-20160811,6600,6000
+20170113,5940,5400
+20170303,5280,4800
+20170331,4620,4200
+20170601,4345,3950
+20170811,4675,4250
+20170908,4950,4500
```

Comment:
Missing entry for `2017-09-22`

Entry should be:
```
20170922,5390,4900
```

Will make entry below a duplicate